plaso/engine/artifact_filters.py

```diff
@@ -194,7 +233,7 @@ def BuildFindSpecsFromFileArtifact(
 
     return find_specs
 
-  def BuildFindSpecsFromRegistryArtifact(self, source_key_path):
+  def _BuildFindSpecsFromRegistrySource(self, source_key_path):
```

Comment:
Technically this method neither uses a Registry artifact or artifact registry source. Maybe:

`_BuildFindSpecsFromRegistryKeyPath` or `_BuildFindSpecsFromWindowsRegistryKeyPath`

Reply:
Renamed to _BuildFindSpecsFromRegistrySourceKey to match the artifacts naming: https://github.com/ForensicArtifacts/artifacts/blob/master/artifacts/source_type.py#L261 https://github.com/ForensicArtifacts/artifacts/blob/master/artifacts/source_type.py#L330